docs/assets/operators.json

```
@@ -0,0 +1,128 @@
+        "https://walrus-testnet-publisher.stakecraft.com": {},
+        "https://pub.test.walrus.eosusa.io": {},
+        "https://walrus-pub.testnet.obelisk.sh": {},
+        "https://walrus-testnet-publisher.stakingdefenseleague.com": {},
+        "https://walrus-testnet.thepassivetrust.com": {},
+        "https://walrus-testnet-publisher.natsai.xyz": {},
+        "https://walrus.testnet.publisher.stakepool.dev.br": {},
+        "https://publisher.walrus.banansen.dev": {},
+        "https://testnet.publisher.walrus.silentvalidator.com": {},
+        "https://testnet-publisher.walrus.graphyte.dev": {},
+        "https://walrus-testnet-publisher.imperator.co": {},
+        "https://walrus-testnet-publisher.unemployedstake.co.uk": {},
+        "https://publisher.walrus-01.tududes.com": {},
+        "https://walrus-publisher-testnet.n1stake.com": {},
+        "https://suiftly-testnet-pub.mhax.io": {},
+        "https://walrus-testnet-publisher.trusted-point.com": {},
+        "https://walrus-testnet-publisher.veera.com": {},
+        "https://publisher.testnet.walrus.atalma.io": {},
+        "https://153-gb3-val-walrus-publisher.stakesquid.com": {},
+        "https://sui-walrus-testnet.bwarelabs.com/publisher": {},
+        "https://walrus-testnet.chainbase.online/publisher": {},
+        "https://walrus-testnet.blockscope.net:11444": {},
+        "https://walrus-publish-testnet.chainode.tech:9003": {},
+        "https://walrus-testnet-publisher.starduststaking.com": {},
+        "http://walrus-publisher-testnet.overclock.run:9001": {},
+        "https://walrus-testnet-publisher.everstake.one": {},
+        "http://walrus.testnet.pops.one:9001": {},
+        "http://ivory-dakar-e5812.walrus.bdnodes.net:9001": {},
+        "http://publisher.testnet.sui.rpcpool.com:9001": {},
+        "http://walrus.krates.ai:9001": {},
+        "http://walrus-publisher-testnet.latitude-sui.com:9001": {},
+        "http://walrus-tn.juicystake.io:9090": {},
+        "http://walrus.globalstake.io:9001": {},
+        "http://walrus-testnet.staking4all.org:9001": {},
+        "http://walrus-testnet.rpc101.org:9001": {},
+        "http://walrus-publisher-testnet.cetus.zone:9001": {},
+        "http://93.115.27.108:9001": {},
+        "http://65.21.139.112:9001": {},
+        "http://162.19.18.19:9001": {},
+        "http://walrus-publisher.stakeme.pro:9001": {},
+        "http://walrus-storage.testnet.nelrann.org:9001": {},
+        "http://walrus-testnet.senseinode.com:9001": {},
+        "http://walrus-testnet.equinoxdao.xyz:9001": {},
+        "https://walrus-testnet-publisher.stakely.io": {},
+        "https://walrus-testnet-publisher.talentum.id": {},
+        "http://37.27.230.228:9001": {},
+        "https://walrus-testnet-published.luckyresearch.org": {},
+        "http://walrus-testnet.suicore.com:9001": {},
+        "https://walrus-testnet.validators.services.kyve.network/publish": {},
+        "http://walrus-publisher-testnet.suisec.tech:9001": {}
+    }
+}
```

Nit: Can we add a final new line?
```suggestion
}

```